lib/gui/models/store.js

```diff
@@ -268,9 +269,9 @@ const storeReducer = (state, action) => {
       });
 
       return _.attempt(() => {
-        if (_.some([
-          selectedDrive && selectedDrive.get('size', 0) < action.data.size,
-          selectedDrive && selectedDrive.get('size', 0) < action.data.recommendedDriveSize
+        if (selectedDrive && _.some([
```

Comment:
This can be refactored to

```js
if (selectedDrive && !_.every([
  constraints.isDriveLargeEnough(selectedDrive.toJS(), action.data),
  constraints.isDriveSizeRecommended(selectedDrive.toJS(), action.data)
]))
```

So we avoid negating in each part of `_.some`.